app/views/accounts/show.html.erb

```diff
@@ -20,39 +20,21 @@
         <% end %>
       </ul>
     <% end %>
+  <!-- Subscriber Account Page  -->
   <% elsif current_user.try(:subscribed) %>
     <h2>Thank you for your support!</h2>
     <p>Feel free to <%= link_to 'watch any video on the site', posts_path %>.  Or, if you're done with learning from me, you can <%= link_to 'unsubscribe', unsubscribe_path, method: :post, data: {confirm: "You hit the 'unsubscribe' link. Hitting OK means you'll no longer have access to Pro videos.  OK?"} %>.</p>
 
     <hr>
-
     <%= link_to "Change your email or password", edit_user_registration_path %>
     <%= render 'shared/change_credit_card' %>
     <hr>
     <a href="/organization_admin#/new" class="btn btn-primary light-button">I want to create an organization from this account.</a>
 
     <% content_for :title, "My Account- EmberScreencasts" %>
+  <!-- Anonymous Sales Page -->
   <% else %>
-    <h1>The best resource for intermediate Ember developers</h1>
-    <p>For just $9.95/month, you get <em>instant</em> <em>on-demand</em> access to every single video in the emberscreencasts library.</p>
-
-    <ul>
-        <li>New up-to-date videos every week.</li>
-        <li>Over 7 hours of fast-paced content.</li>
-        <li>No commitment- unsubscribing is instant and easy.</li>
-    </ul>
-
-    <h2>Save time.  Learn more.  Be better.</h2>
-    <%= render 'shared/subscribe' %>
-    <br>
-    <a href='https://www.expeditedssl.com/simple-ssl-scanner/scan?target_domain=www.emberscreencasts.com' target='_blank'><img src='https://www.expeditedssl.com/ssl-secure-badge.png'  alt='Expedited SSL Scanner'></a>
-    <br>
-    <p>Buying for your organization?  You can add more users to your account after signing up.</p>
-
-    <!-- Lots of testimonials -->
-    <!-- Another subscribe button -->
-
-    <% content_for :title, "Learn like a Pro- sign up for an EmberScreencasts Pro Subscription." %>
+  <%= controller.redirect_to sales_path %>
```

Comment:
This is an unusual construct- typically I would expect to see this redirect in the controller.  Have conventions changed around this recently?